packages/palette/src/elements/Message/Message.tsx

```diff
@@ -34,7 +47,7 @@ export const Message: FC<MessageProps> = ({
 }) => {
   return (
     <StyledFlex p={2} {...others}>
-      <Sans size={textSize} color={color("black60")} weight="regular">
+      <Sans size={textSize} color={color("black100")} weight="regular">
```

Comment:
Should this be `black100` for all three variants, or just for `info` (because of the blue)?

Reply:
good q! `black100` for all variants https://artsy.slack.com/archives/C9ZJYFDNF/p1605820747449100 / https://www.figma.com/file/LKKhtse11fjmdFI6AaVqGM/Patricia's-rough-bug-Palette?node-id=1844%3A3294